.github/workflows/build-test.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        node-version: [12.x]
+        node-version: [14.x, 16.x]
```

Comment:
This package still supports Node.js v12 (according to `package.json`), so we should continue to test it until we drop support

Reply:
Good catch! Fixed in [29f4994](https://github.com/MetaMask/eth-block-tracker/pull/97/commits/29f499475854291e673ce72d861d715c38c32143).